ceph.spec.in

```diff
@@ -1349,7 +1353,13 @@ cmake .. \
     -DSYSTEMD_SYSTEM_UNIT_DIR:PATH=%{_unitdir} \
     -DWITH_MANPAGE:BOOL=ON \
     -DWITH_PYTHON3:STRING=%{python3_version} \
+%if 0%{with mgr_dashboard_frontend_support_multi_language}
+    -DWITH_MGR_DASHBOARD_FRONTEND:BOOL=ON \
+    -DDASHBOARD_FRONTEND_LANGS:STRING="cs,de,es,fr,id,it,ja,ko,pl,zh-Hans,zh-Hant,pt" \
```

Comment:
2 suggestions here:
* Make this the default behaviour for the build process (otherwise this will have be added to the Debian/Ubuntu builds too).
* Isn't there a way to tell it to build ALL languages? If there are certain translations that don't meet a minimum quality (or percentage of strings translated), we can do the other way around: add an explicit list of excluded translations.

Reply:
Good ideas, I will set default value of `DASHBOARD_FRONTEND_LANGS ` to 'ALL' in CMakeLists.txt and set `WITH_MGR_DASHBOARD_FRONTEND` to on in ceph.spec to resovle the above two suggestions.